### ISSUE TITLE
Error on build when using ES2015

### ISSUE THREAD

**Issue by bwhitty** [xolvio/meteor-jasmine#283]
```shell
W20150925-13:08:58.549(-4)? (STDERR) [sanjo:jasmine]: The file "/x/y/z/server/bootstrap.js" has syntax errors. [SyntaxError: Unexpected token )]
W20150925-13:08:58.550(-4)? (STDERR) [sanjo:jasmine]: The file "/x/y/z/server/publish.js" has syntax errors. [SyntaxError: Unexpected token >]
```

The above displays in the console where `meteor run` was executed each time the app is rebuilt. I just updated to `0.19.1`, but it still is happening. Oddly enough, the mirrors seem to be running tests (which are written in ES2015) fine and `html-reporter` is still updating fine.

**Comment by Sanjo** [comment]
This probably comes from the server unit mode that doesn't support ES2015. You can disable it with `process.env.JASMINE_SERVER_UNIT = 0`. Having no server unit tests should also do it I think. If have have some, you can move them to the server/integration folder.

**Comment by bwhitty** [comment]
Running with `JASMINE_SERVER_UNIT=0` env did alleviate the issue, but the former solution did not. I had no tests inside a `unit/` folder, and the error was still persistent.

**Comment by Sanjo** [comment]
Fixed with 0.20.0. I have disabled the server unit mode by default as it doesn't work with ES2015. See https://github.com/xolvio/meteor-jasmine/commit/2e52e6afb7faf22eb5349f15c0e2532b5f63bc51
